antslib/configer.py

```diff
@@ -11,6 +11,12 @@
 import grp
 import re
 
+try:
+    from antslib import macos_prefs
+    use_macos_prefs = True
```

Comment:
Because the variable `use_macos_prefs` is only defined when the import succedes, we can end up with a state in which the variable is undefined. This can cause the following error:
```
Traceback (most recent call last):
  File "/private/tmp/env_ants/bin/ants", line 28, in <module>
    from antslib import logger
  File "/private/tmp/env_ants/lib/python2.7/site-packages/antslib/logger.py", line 16, in <module>
    CFG = configer.read_config('main')
  File "/private/tmp/env_ants/lib/python2.7/site-packages/antslib/configer.py", line 67, in read_config
    if use_macos_prefs:
NameError: global name 'use_macos_prefs' is not defined
```
I suggest setting `use_macos_prefs = False` ahead of the try statement.

Reply:
Nice catch (no pun intended). Will fix later today. Out of curiousity, is there any reason not to have `use_macos_prefs = False` replace the `pass` statement below `except ImportError`? Or should it go above the try statement?

Reply:
Good point. That's even nicer.